front/src/modules/ui/navbar/components/NavItem.tsx

```diff
@@ -81,7 +82,28 @@ const StyledSoonPill = styled.div`
   padding-right: ${({ theme }) => theme.spacing(2)};
 `;
 
-function NavItem({ label, icon, to, onClick, active, danger, soon }: OwnProps) {
+const StyledItemCount = styled.div`
```

Comment:
Broken on light theme : 

<img width="241" alt="image" src="https://github.com/twentyhq/twenty/assets/26528466/4e7e04fe-d3f1-45d7-aca5-5c0bcef1cf5c">

The font color should be white no ?